modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -72,7 +73,7 @@ import WindowManager from './WindowManager';
 export interface EditorConstructor {
   readonly prototype: Editor;
 
-  new (id: string, settings: RawEditorSettings, editorManager: EditorManager): Editor;
+  new (id: string, settings: RawEditorOptions, editorManager: EditorManager): Editor;
```

Comment:
Should this be `options`?

```suggestion
  new (id: string, options: RawEditorOptions, editorManager: EditorManager): Editor;
```

Reply:
It should be thanks :+1: